tests/php/sync/test_class.jetpack-sync-meta.php

```diff
@@ -7,102 +7,129 @@ class WP_Test_Jetpack_Sync_Meta extends WP_Test_Jetpack_Sync_Base {
 	protected $post_id;
 	protected $meta_module;
 
+	protected $whitelisted_post_meta = 'content_width';
+
 	public function setUp() {
 		parent::setUp();
 
 		// create a post
 		$this->meta_module = Jetpack_Sync_Modules::get_module( "meta" );
-		$this->meta_module->set_post_meta_whitelist( array( 'test_meta_key' ) );
 		$this->post_id = $this->factory->post->create();
-		add_post_meta( $this->post_id, 'test_meta_key', 'foo' );
+		add_post_meta( $this->post_id, $this->whitelisted_post_meta, 'foo' );
```

Comment:
Rather than hard-coding a brittle constant value that is currently whitelisted, it might be cleaner and less brittle to whitelist a specific test post meta here, like 'foobar'.